### DISCUSSION TITLE
[get the Rpc from Zksync Node] <how to get the RPC URL from Zksync node>

### DISCUSSION THREAD

**Discussion by aykut9607** [zkSync-Community-Hub/zksync-developers#560] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

guys is there anyone knows how can we get the RPC URL from zksync full node.is there a code to write in the terminal and get the  RPC URL as results.my node is sync and working and all.

**Comment by 0xsisyfos** [comment, marked as answer]
Hey,

You can find more information on how to run an in-memory node for testing in our docs:
https://docs.zksync.io/build/test-and-debug/era-test-node.html#installing-and-setting-up-era-test-node

Your local zkSync Era node is accessible at http://127.0.0.1:8011/.

**Reply by aykut9607** [reply]
> Hey,
> 
> You can find more information on how to run an in-memory node for testing in our docs: https://docs.zksync.io/build/test-and-debug/era-test-node.html#installing-and-setting-up-era-test-node
> 
> Your local zkSync Era node is accessible at http://127.0.0.1:8011/.

hi thanks for repling.my node is full node running on mainnet .are you familiar with getting rpc from zksync full node
